ceno_host/tests/test_elf.rs

```diff
@@ -11,38 +11,62 @@ use itertools::enumerate;
 #[test]
 fn test_ceno_rt_mini() -> Result<()> {
     let program_elf = ceno_examples::ceno_rt_mini;
-    let mut state = VMState::new_from_elf(CENO_PLATFORM, program_elf)?;
+    let program = Program::load_elf(program_elf, u32::MAX)?;
```

Comment:
This snippet is repeated a lot.  TODO: deduplicate.

Reply:
indeed..